src/python/pants/backend/jvm/subsystems/zinc.py

```diff
@@ -383,21 +383,6 @@ def snapshot(self, scheduler):
       ),
     ))[0]
 
-  # TODO: Make rebase map work without needing to pass in absolute paths:
-  # https://github.com/pantsbuild/pants/issues/6434
-  @memoized_property
-  def rebase_map_args(self):
```

Comment:
Is there any code that asserts that all inputs are located under the buildroot? There used to be, but would be good to make sure it is still in place, because failing to rebase something leads to "silent" incremental compile failures.

I began adding additional analysis portability tests in https://github.com/pantsbuild/pants/pull/6254 and need to pick that back up soon most likely. But in the meantime, getting some asserts in place that "everything" we feed to zinc is always below `cwd` would be good.